stages/org.osbuild.yum.repos

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/python3
+"""
+Create YUM / DNF repo file in /etc/yum.repos.d
+
+All repo-specific options are supported. Only a subset of options which can be
+used in a repo or [main] configuration is supported, specifically:
+  - gpgcheck
+  - repo_gpgcheck
```

Comment:
The patch seems to support many more though, no? ;)

Reply:
No. There are options, which can be specified only for a repo and then there are options which can be specified for both the `[main]` section of the global config as well as for a repo. The list lists options which can be specified for both, the repo and `[main]`. As for the options allowed only for a repo configuration, all of them are supported (except the `type` which didn't make much sense and the fact is noted in the doc string).

Does that make sense? I'm happy to reword the comment a bit to make it clearer.

Reply:
I reworded the comment above the list a bit. Hope it is easier to understand... 🤔